Localization/ru-RU.hjson

```diff
@@ -4,5 +4,31 @@ Mods: {
 		TechNamesOff: "[WMITF] Технические названия ВЫКЛ"
 		WorldTooltipsOn: "[WMITF] Названия НИП/тайлов ВКЛ"
 		WorldTooltipsOff: "[WMITF] Названия НИП/тайлов ВЫКЛ"
+
+		Keybinds: {
+			// "Technical Names.DisplayName": Display Technical Names
+			// "Tile/NPC Mod Tooltip.DisplayName": Display Tile/NPC Mod Tooltips
+		}
+
+		Configs: {
+			Config: {
+				// DisplayName: Config
+
+				DisplayWorldTooltips: {
+					// Label: Display World Tooltips
+					// Tooltip: Whether or not mod origins are displayed when hovering over NPCs and Tiles.
+				}
+
+				DisplayItemTooltips: {
+					// Label: Display Items Tooltips
+					// Tooltip: Whether or not mod origins are displayed in item tooltips.
+				}
+
+				DisplayTechnicalNames: {
+					// Label: Display Technical Names
+					// Tooltip: Whether or not technical names are displayed.
```

Comment:
```suggestion
		WorldTooltipsOn: "[WMITF] Подсказки для НИП/блоков ВКЛ"
		WorldTooltipsOff: "[WMITF] Подсказки для НИП/блоков ВЫКЛ"

		Keybinds: {
			"Technical Names.DisplayName": Показывать технические названия
			"Tile/NPC Mod Tooltip.DisplayName": Показывать моды для НИП/блоков
		}

		Configs: {
			Config: {
				DisplayName: Конфигурация

				DisplayWorldTooltips: {
					Label: Показывать подсказки вне инвентаря
					Tooltip: Позволяет увидеть мод, добавивший данный блок или НИПа.
				}

				DisplayItemTooltips: {
					Label: Показывать подсказки в инвентаре
					Tooltip: Позволяет увидеть мод, добавивший данный предмет.
				}

				DisplayTechnicalNames: {
					Label: Показывать технические названия
					Tooltip: Показывает значения Mod.Name, ModItem.Name и пр. вместо названия мода.
```

Reply:
Hmm looks like I can "commit suggestion" right from GitHub's UI. Might as well do that.